src/renderer/components/Validator/ValidatorSimple.tsx

```diff
@@ -12,30 +14,56 @@ export interface IValidatorSimpleProps {
     onRemoveClick: () => void;
     onExportClick: () => void;
     privateKey: string;
+    nodes: BeaconNode[];
 }
 
 export const ValidatorSimple: React.FunctionComponent<IValidatorSimpleProps> = (
     props: IValidatorSimpleProps) => {
 
+    const renderBeaconNodes = (): React.ReactElement => (
+        <div className="validator-nodes">
+            <div className="node-container">
+                <div className="node-grid-container" >
+                    {props.nodes.map(node => (
+                        <NodeCard
+                            key={node.url}
+                            onClick={(): void=>{}}
+                            title="Beacon node"
```

Comment:
I was thinking, maybe we can put "<container name> docker container" and "Remote beacon node" depending on what type it is?